lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1029,6 +1029,9 @@ fc_extras
         try:
             iris.unit.as_unit(attr_units)
         except ValueError:
+        # formatting a unicode string causes an exception. Test for that.
+            if isinstance(attr_units, unicode):
+                attr_units = attr_units.encode('ascii', errors='backslashreplace')
```

Comment:
So, a stack of tests aren't running on Master? To be investigated. This is however now passing (I checked that the test ran, and things look ok!) In Python3, the all-strings-are-unicode change will render this moot, and unicode-containing NetCDF would be automagically compatible (and this change should be stripped out at that time).